packages/neon/neon_files/lib/neon_files.dart

```diff
@@ -1,99 +1 @@
-/// The Neon client for the files app.
-///
-/// Add `FilesApp()` to your runNeon command to execute this app.
-///
-/// A basic implementation could look like:
-///```dart
-///Future<void> main() async {
-///  await runNeon(
-///    appImplementations: {
-///      FilesApp()
-///    },
-///    theme: NeonTheme(
-///      branding: Branding(
-///       name: 'Files',
-///       logo: VectorGraphic(
-///         loader: AssetBytesLoader(
-///           'assets/logo.svg.vec',
-///         ),
-///       ),
-///      ),
-///    ),
-///  );
-/// }
-///```
-library files;
```

Comment:
We should still keep the library declaration (iirc dart doc will complain if they are not present).

Reply:
Ok, but I would like to remove the example because I fear it will get outdated very fast and we will take a long time to notice. I also don't think it is particularly helpful, we should rather point to the neon package in the README.md and explain how to use the package.

Reply:
Ok. The first line should be kept though.